src/utils/error.ts

```diff
@@ -49,6 +52,17 @@ export abstract class CliKintoneError extends Error {
     }
   }
 
+  private _networkError(error: any): boolean {
+    return error && error.code === "ECONNABORTED";
+  }
+
+  private _toStringNetworkError(error: any): string {
+    let errorMessage = `[${error.code}] ${error.message}\n`;
+    errorMessage += "The cli-kintone aborted due to a network error.\n";
+    errorMessage += "Please check your network connection.\n";
+    return errorMessage;
+  }
+
```

Comment:
IMO: It's better to use a user-defined type guard to narrowing type.
https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates

```suggestion  
  private _networkError(
    error: unknown
  ): error is { code?: string; message?: string } {
    return (
      typeof error === "object" &&
      error !== null &&
      "code" in error &&
      error.code === "ECONNABORTED"
    );
  }

  private _toStringNetworkError(error: {
    code?: string;
    message?: string;
  }): string {
    let errorMessage = `[${error.code}] ${error.message}\n`;
    errorMessage += "The cli-kintone aborted due to a network error.\n";
    errorMessage += "Please check your network connection.\n";
    return errorMessage;
  }
```

It's also good to define a private type.

```
type NetworkError = { code?: string; message?: string };
```